tests/Team.py

```diff
@@ -114,11 +116,15 @@ def testRepos(self):
         repo = self.org.get_repo("FatherBeaver")
         self.assertListKeyEqual(self.team.get_repos(), None, [])
         self.assertFalse(self.team.has_in_repos(repo))
+        self.assertIsNone(self.team.get_repo_permission(repo))
         self.team.add_to_repos(repo)
         self.assertListKeyEqual(
             self.team.get_repos(), lambda r: r.name, ["FatherBeaver"]
         )
         self.assertTrue(self.team.has_in_repos(repo))
+        self.assertIsInstance(
+            self.team.get_repo_permission(repo), github.Permissions.Permissions
+        )
```

Comment:
I'd rather you assert something in the returned object, such as:

```
permissions = self.team.get_repo_permission(repo)
self.assertTrue(permissions.pull)
```

Reply:
Sounds good, that also drops the need for the import.  I've force-pushed (first time I used github UI to resolve the Const conflict but now I've rebased and fixed it there instead)